pandas/tests/io/excel/test_readers.py

```diff
@@ -439,6 +445,9 @@ def test_bad_engine_raises(self, read_ext):
 
     @tm.network
     def test_read_from_http_url(self, read_ext):
+        if read_ext == '.ods':  # TODO: remove once on master
```

Comment:
This test only works when the file is available on master, so have to merge first and then can try again

Reply:
Seems like this code is still hanging around ill aim to address it when I tackle: https://github.com/pandas-dev/pandas/issues/29439